app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -51,4 +53,16 @@ private User findUser(Long id) {
         return userRepository.findByIdAndDeletedIsFalse(id)
                 .orElseThrow(() -> new UserNotFoundException(id));
     }
+
+    public User findUserByEmailAndPassword(final String email, final String password) {
+        final User foundUser = userRepository.findByEmail(email)
+                .orElseThrow(() -> new UserEmailNotFoundException("찾을 수 없는 이메일 입니다."));
```

Comment:
이렇게 할 수도 있겠죠. 이 예외 메시지를 읽는 사람이 새벽 3시에 디버깅하고 있는 미래의 자신이라고 생각해 보세요.

```suggestion
                .orElseThrow(() -> new UserEmailNotFoundException(email + "은 찾을 수 없는 이메일 입니다."));
```

Reply:
넵  예외 메시지도 상세하게 써보겠습니다!